Classes/Issues/GithubClient+Issues.swift

```diff
@@ -297,6 +298,37 @@ extension GithubClient {
         }
     }
 
+    func setSubscription(
+        previous: IssueResult,
+        subscribe: Bool,
+        completion: ((Result<SubscriptionState?>) -> Void)? = nil
```

Comment:
Hm, what would it mean if the `state` is `nil`? Would that be an actual success?